client/src/ErrorAlert/ErrorAlert.js

```diff
@@ -19,18 +22,79 @@ ContactUs.propTypes = {
   message: PropTypes.string.isRequired
 }
 
+const showConfirmationAlert = email => <ConfirmationAlert email={email} />
 const resendToken = type => <ResendToken type={type} />
 const contactUs = message => <ContactUs message={message} />
 
-export const ErrorAlert = ({ attribute, type }) => {
+const ConfirmationAlert = ({ email }) => {
+  const { t } = useTranslation()
+  const history = useHistory()
+  const { makeRequest } = useApiResponse()
+
+  const resendConfirmation = async () => {
+    const response = await makeRequest({
+      type: 'post',
+      url: '/confirmation',
+      data: {
+        email
+      }
+    })
+    if (response.ok) {
+      history.push({
```

Comment:
This is a hacky solution to close the error alert and display the "Email resent" success message on the login page (the confirmation link is displayed in an alert). If you have a better solution, do let me know!